.emacs.d/lisp/package/git-update.el

```diff
@@ -0,0 +1,78 @@
+;;; THIS FILE IS NOT LOADED AT STARTUP.
+;;; This file is autoloaded on the following symbols:
+;;;  kotct/update-dot-config
+;;;  kotct/update-user-config
+
+(require  'user-config-system)
+
+(package-initialize)
+
+(defun kotct/git-repository-diff (directory)
+  "Fetches in DIRECTORY and returns the difference between the common ancestor of HEAD and origin/master in DIRECTORY."
+  (let ((default-directory directory))
+    (kotct/run-git "fetch" "origin" "master")
+    (kotct/run-git "diff" "HEAD...origin/master")))
+
+(defun kotct/git-repository-is-behind (directory)
+  "Fetch DIRECTORY's origin/master and diff to see if there are any new remote commits."
+  (not (string= (kotct/git-repository-diff directory) "")))
+
+(defun kotct/user-fetch-config (username)
+  "Fetch USERNAME's personal config from GitHub, out of the
+repository USERNAME/.emacs."
```

Comment:
Lots of other docstrings don't wrap until the end of the sentence; is that not desired here too?

Reply:
Sorry this was just naively copied. see: https://github.com/kotct/dot/pull/125/files/443852b982f5394d004e9548c785ca2b3200612b#diff-b66ea82accaeecea8494ccaa90aec31fL39
done.